### ISSUE TITLE
publication with multiple publication-entries in 260 and 264

### ISSUE THREAD

**Issue by TobiasNx** [hbz/lobid-resources#1886]
In Aleph there seems to be only one publication entry for the primary "PublicationEvent".

But Marc allows for multiple including the historical publishers.
Also there are publisher infos in element 260 and 264.

[260 - Publication, Distribution, etc. (Imprint) (R) ](https://www.loc.gov/marc/bibliographic/bd260.html)
[264 - Production, Publication, Distribution, Manufacture, and Copyright Notice (R)](https://www.loc.gov/marc/bibliographic/bd264.html)

"Information in field 260 is similar to information in field 264 (Production, Publication, Distribution, Manufacture, and Copyright Notice). Field 260 is useful for cases where the content standard or institutional policies used do not make a distinction between functions."

"Information in field 264 is similar to information in field 260 (Publication, Distribution, etc. (Imprint)). Field 264 is useful for cases where the content standard or institutional policies make a distinction between functions."

MARC also allows to state historical publication too, marced via the first indicator:

```
First Indicator 
Sequence of statements
# - Not applicable/No information provided/Earliest
2 - Intervening
3 - Current/Latest
```

This seems to create problems especially with periodicals since they often have multiple infos about the publication.

The Aleph Transformation merged distinct information in a single publication object in the arrray.
http://aleph.lobid.org/resources/HT001387709

```
"publication": [
    {
      "type": [
        "SecondaryPublicationEvent"
      ],
      "description": [
        "Hamburg : Alphacom. - Mikrofiche-Ausg.: Wohltorf : IOS-Echo-GmbH. - Mikrofiche-Ausg.: Berlin : ADN. - Mikrofilm-Ausg.: Woodbridge, Conn. : Primary Source Media, 2000. Mikrofilme ; 16 mm",
        "Mikrofiche-Ausg."
      ],
      "location": [
        "Hamburg",
        "Mikrofiche-Ausg."
      ],
      "publishedBy": [
        "Alphacom",
        "Mikrofiche-Ausg."
      ],
      "startDate": "2000"
    },
    {
      "type": [
        "PublicationEvent"
      ],
      "frequency": [
        {
          "id": ["http://marc21rdf.info/terms/continuingfre#w"](http://marc21rdf.info/terms/continuingfre#w),
          "label": "wöchentlich"
        }
      ],
      "note": [
        "Jahrgangszählung ab 71. Jahrgang, Heft 13 (24. März 2018); 71. Jahrgang, Heft 19-72. Jahrgang, Heft 18 in der Zählung ausgelassen; Periodizität: wöchentl."
      ],
      "publicationHistory": "1.1946,1(16.Nov.)-6[?]; 1.1947,1(4.Jan.) - 52.1998; 1999 -",
      "location": [
        "Hannover",
        "Hamburg"
      ],
      "publishedBy": [
        "Spiegel-Verl. Augstein",
        "Spiegel-Verlag Rudolf Augstein"
      ],
      "startDate": "1946"
    }
  ],
```

The ALMA transformation creates multiple objects for each publication info.

http://lobid.org/resources/990054089950206441

```
"publication": [
    {
      "publicationHistory": "1.1946,1(16.Nov.)-6[?]; 1.1947,1(4.Jan.) - 52.1998; 1999 -",
      "startDate": "1946",
      "type": [
        "PublicationEvent"
      ],
      "location": [
        "Hamburg"
      ],
      "publishedBy": [
        "Spiegel-Verlag Rudolf Augstein"
      ],
      "frequency": [
        {
          "id": ["http://marc21rdf.info/terms/continuingfre#w"](http://marc21rdf.info/terms/continuingfre#w),
          "label": "wöchentlich"
        }
      ],
      "note": [
        "Jahrgangszählung ab 71. Jahrgang, Heft 13 (24. März 2018); 71. Jahrgang, Heft 19-72. Jahrgang, Heft 18 in der Zählung ausgelassen; Periodizität: wöchentl."
      ]
    },
    {
      "publicationHistory": "1.1946,1(16.Nov.)-6[?]; 1.1947,1(4.Jan.) - 52.1998; 1999 -",
      "type": [
        "PublicationEvent"
      ],
      "location": [
        "Hannover"
      ],
      "frequency": [
        {
          "id": ["http://marc21rdf.info/terms/continuingfre#w"](http://marc21rdf.info/terms/continuingfre#w),
          "label": "wöchentlich"
        }
      ],
      "note": [
        "Jahrgangszählung ab 71. Jahrgang, Heft 13 (24. März 2018); 71. Jahrgang, Heft 19-72. Jahrgang, Heft 18 in der Zählung ausgelassen; Periodizität: wöchentl."
      ]
    },
    {
      "publicationHistory": "1.1946,1(16.Nov.)-6[?]; 1.1947,1(4.Jan.) - 52.1998; 1999 -",
      "type": [
        "PublicationEvent"
      ],
      "location": [
        "Hamburg"
      ],
      "publishedBy": [
        "Spiegel-Verl. Augstein"
      ],
      "frequency": [
        {
          "id": ["http://marc21rdf.info/terms/continuingfre#w"](http://marc21rdf.info/terms/continuingfre#w),
          "label": "wöchentlich"
        }
      ],
      "note": [
        "Jahrgangszählung ab 71. Jahrgang, Heft 13 (24. März 2018); 71. Jahrgang, Heft 19-72. Jahrgang, Heft 18 in der Zählung ausgelassen; Periodizität: wöchentl."
      ]
    },
    {
      "type": [
        "SecondaryPublicationEvent"
      ],
      "location": [
        "Hamburg",
        "Wohltorf",
        "Berlin",
        "Woodbridge, Conn."
      ],
      "description": [
        "Mikrofiche-Ausg."
      ],
      "publishedBy": "Primary Source Media"
    }
  ],
```
Also Info from 260 and 264 can be redundant.
There are two things to decide and to do here:
1) Should we always prefer 264 info over 260 if info is given? 
2) Should we keep the historical publishing info in ALMA and not merge them in one element?
3) How do we handle the info that is given not in 264 or 260 but from other marc element as  frequency (`006`,`008`), publicationNote (`515`), publicationDates from `500`. Should we only add them to the current publication object?

My suggestion:
1) Yes we should. Perhaps we can use a fallback mechanism for 260 when info as pubdate is missing in 264.
2) Yes we should.
3) Only add  this info to the current/latest publisher entry.

**Comment by TobiasNx** [comment]
Talked to @acka47 off board so that we can proceed here.

The preference of 264 over 260 is supported by:
https://www.alma-dach.org/alma-marc/bibliographic/260/260.html
https://www.alma-dach.org/alma-marc/bibliographic/264/264.html

**Comment by TobiasNx** [comment]
This seems to work and have reduced the redundancy: 

http://lobid.org/resources/990054089950206441

GUI could be improved by only showing in published element.
Also related problems need to be solved in #1899.

**Comment by acka47** [comment]
Closing
